00-programming-fundamentals/04-installfest.md

```diff
@@ -70,3 +70,14 @@ $ git config --global color.branch auto
 There are many options for text editors. _Sublime Text_, _TextMate_ and _Atom_ are popular **Graphic User Interface (GUI)** editors. _vim_ and _emacs_ are popular editors that operate directly on the command line.
 
 **You should have already installed a text editor in Jump Start, but if not, please install [Atom](https://atom.io/) now.**
+
+## It's All About Communication!
+
+Lastly we you should already have Ada's primary mode of communication installed... [**Slack!**](https://slack.com/downloads/osx).  While it is **possible** to get by using the Slack website.  We recommend **strongly** to use the Desktop client.  Simply follow the link above and install it, or go to the Mac Appstore and install Slack that way.  
+
+When Ada has announcements or students want to share general information we generally use Slack.  We only use e-mail for personal communication, things we need to keep a record of (like absences).  
```

Comment:
I'd just remove this last sentence since Ada Staff often e-mails students for other things